src/pages/STG_PanelRelAuto.page

```diff
@@ -1,15 +1,39 @@
 <apex:page controller="STG_PanelRelAuto_CTRL" docType="html-5.0" standardStylesheets="false" >
+    <apex:includeLightning />
+
+    <script>
+        if(!{!hasAccess}) {
+            $Lightning.use("c:RD2_EnablementApp", function() {
+                $Lightning.createComponent("c:utilIllustration",
+                    {   title : "{!$Label.commonAdminPermissionErrorTitle}",
+                        message : "{!$Label.commonPermissionErrorMessage}",
+                        size: 'large',
+                        variant: 'no-access'
+                    },
+                    "utilIllustration",
```

Comment:
@bdvorachek - please note that I had to update the element id here and below as they were conflicting with the other npsp settings pages (technically all in one page). Just want to make sure we don't revert it back to this one once Daniel's PR is merged in.

Thanks!

Reply:
Will check once #5583 is merged. Not all contents of that PR are in this PR, some of it will appear once MetaCI's auto merge happens.